packages/material-ui/src/Link/Link.js

```diff
@@ -95,23 +133,27 @@ const Link = React.forwardRef(function Link(props, ref) {
     }
   };
 
+  const styleProps = {
+    ...other,
```

Comment:
```suggestion
    ...props,
```

The `styleProps` should contain **all props** plus their defaults. That's why we are spreading first the props, and then adding all calculated props, or props that have some default value.